test/tests/functional/pbs_qmgr.py

```diff
@@ -159,3 +208,143 @@ def test_multi_attributes(self):
             self.logger.info("looking for error, %s" % msg)
             self.assertTrue(msg in e.msg[0])
         self.server.expect(NODE, 'queue', op=UNSET, id=self.mom.shortname)
+
+    @tags('multisched')
+    def test_list_sched(self):
+        """
+        Test to verify that qmgr list sched works for multiple schedulers
+        """
+        # Scenario 1: only default sched present
+        # Scenario 2: multiple schedulers present
```

Comment:
These comments should be part of the doc string.  Also if you are doing two distinct scenarios in one test, it is probably better to make two tests.